### PR TITLE
gui/settings dialog: Add cpu backend choice.

### PR DESCRIPTION
**Pull request by Zangetsu38**
# About
- Add cpu backend choice in setting dialog.
- Add cpu backend used in log.
- reafctor cpu backend in app init

# Result:
image

### CODE REVIEW

File: vita3k/host/include/host/state.h [hunk 1]
```diff
@@ -79,6 +79,7 @@ struct HostState {
     uint32_t fps = 0;
     uint32_t ms_per_frame = 0;
     WindowPtr window = WindowPtr(nullptr, nullptr);
+    CPUBackend cpu_backend;
```
Comment (sunho) [inline vita3k/host/include/host/state.h:82]: No please don't do this

Comment (Zangetsu38) [reply vita3k/host/include/host/state.h:82]: why ? i need this

Comment (Zangetsu38) [reply vita3k/host/include/host/state.h:82]: > 
> 
> Just use host.kernel.cpu_backend

oh hmm i check sorry

Comment (Zangetsu38) [reply vita3k/host/include/host/state.h]: fixed sorry, i have don't see this value exist

**Review by sunho — APPROVED** [review]
Thanks for doing this. Love your ui.

There's a little bug that the backend change for callback pool apply after restarting emu. I'll deal with this in another pr.

**Pull request merged**